recipe/meta.yaml

```diff
@@ -36,12 +35,16 @@ test:
     - pip
```

Comment:
Could you run pytest as the `tests` folder is present in the census module? 
`- pytest -v --pyargs census  `

Reply:
Most of the tests call an endpoint and after a few calls the server rejects the calls with "too many requests". 

Either we call 1 test of that kind only (per platform, per python version).

Despite all of this, a smoke test is welcome :)